easybuild/easyblocks/g/gromacs.py

```diff
@@ -201,14 +238,48 @@ def configure_step(self):
                 run_cmd(plumed_cmd, log_all=True, simple=True)
 
         else:
+            if '-DGMX_MPI=ON' in self.cfg['configopts']:
+                if self.cfg.get('mpi_numprocs', 0) == 0:
```

Comment:
I would grab `mpi_numprocs` from `self.cfg` once, which cleans things up a bit below (and lowers the risk for introducing bugs by fat fingering on `'mpi_numprocs'`):

```python
mpi_numprocs = self.cfg.get('mpi_numprocs', 0)
if mpi_numprocs == 0:
    ...
    mpi_numprocs = self.cfg['parallel']
...

self.cfg.update('configopts', "-DNUMPROC=%s" % mpi_numprocs)

Reply:
Done